code/modules/projectiles/guns/ds13/pulse_rifle.dm

```diff
@@ -12,7 +12,7 @@ The Pulse Rifle is the standard-issue service rifle of the Earth Defense Force a
 	w_class = ITEM_SIZE_HUGE
 	handle_casings = CLEAR_CASINGS
 	magazine_type = /obj/item/ammo_magazine/pulse
-	allowed_magazines = /obj/item/ammo_magazine/pulse
```

Comment:
does this not work on subtypes? It really should

Reply:
It doesn't. It's been reported a few times that HV mags cannot be inserted.